### DISCUSSION TITLE
Test suite set up methods

### DISCUSSION THREAD

**Discussion by savansonii** [sqlalchemy/sqlalchemy#7866] [Usage Questions]
Hello Everyone,

I wanted to execute pandas test suite with oracle database, pandas uses SQLAlchemy to connect to oracle.
There are test set up functions in pandas tests which are common for multiple databases as they all uses sqlalchemy. 

But they are all failing for oracle.

I wanted to know if I have to set up anything like parameter or configuration from SQLAlchemy to make it run for oracle?

**Comment by CaselIT** [comment]
Hi,

We do not manage the pandas test suite, so we cannot offer help on it. You could try asking to the pandas developers

**Reply by savansonii** [reply]
Ok Thanks!
